include/tvTypesODM.h

```diff
@@ -58,105 +58,6 @@ extern "C"
-
-typedef short int tvPictureMode_t;          //!< Picture mode struct variable 
-
-
-/**
- * @brief Enumeration defining supported HDR video formats
- */
-typedef enum
-{
-    tvVideoHDRFormat_NONE= 0x00,                  //!< No video format 
-    tvVideoHDRFormat_SDR = 0x01,                  //!< video format is SDR 
-    tvVideoHDRFormat_HLG = 0x02,                  //!< video format is HLG 
-    tvVideoHDRFormat_HDR10 = 0x04,                //!< video format is HDR10 
-    tvVideoHDRFormat_HDR10PLUS = 0x08,            //!< video format is HDR10plus 
-    tvVideoHDRFormat_DV = 0x010,                  //!< video format is DV
-    tvVideoHDRFormat_MAX = 0x11                   //!< End of enum
-}tvVideoHDRFormat_t;
-    
-#ifndef __VE_HDR_TYPE__
-#define __VE_HDR_TYPE__
-/** 
- * @brief Enumeration defining supported hdr types
- */
-typedef enum tvhdr_type_e
-{
-    HDR_TYPE_NONE      = 0,                     //!< No HDR Type
-    HDR_TYPE_HDR10     = 1,                     //!< HDR Type is HDR10
-    HDR_TYPE_HDR10PLUS = 2,                     //!< HDR Type is HDR10PLUS
-    HDR_TYPE_DOVI      = 3,                     //!< HDR Type is DOVI
-    HDR_TYPE_PRIMESL   = 4,                     //!< HDR Type is PRIMESL
-    HDR_TYPE_HLG       = 5,                     //!< HDR Type is HLG
-    HDR_TYPE_SDR       = 6,                     //!< HDR Type is SDR
-    HDR_TYPE_MVC       = 7,                     //!< HDR Type is MVC
-    HDR_TYPE_MAX                                //!< End of enum
-} tvhdr_type_t;
-#endif
-
-// ODM Specific. To be Depricated
-typedef enum {
-    tvContentFormatType_NONE= 0x00,
-    tvContentFormatType_SDR = 0x01,
-    tvContentFormatType_HLG = 0x02,
-    tvContentFormatType_HDR10 = 0x03,
-    tvContentFormatType_HDR10PLUS = 0x04,
-    tvContentFormatType_DOVI = 0x05,
-    tvContentFormatType_MAX
-}tvContentFormatType_t;
-
-//ODM specific. To be Depricated
-/** 
- * @brief Enumeration defining supported component color types
- */
-typedef enum
-{
-    COLOR_NONE  = 0,                            //!< Color is None
-    COLOR_RED,                                  //!< Color is Red
-    COLOR_GREEN,                                //!< Color is Green
-    COLOR_BLUE,                                 //!< Color is Blue
-    COLOR_CYAN,                                 //!< Color is Cyan
-    COLOR_MAGENTA,                              //!< Color is Megenta
-    COLOR_YELLOW,                               //!< Color is Yellow
-    COLOR_MAX                                   //!< End of enum
-}tvcomponent_color_type_t;
-
-//ODM specific. To be Depricated
-/**
- * @brief Enumeration defining supported component states
- */
-typedef enum
-{
-    COMPONENT_DISABLE = 0,                      //!< State is Disable
-    COMPONENT_ENABLE,                           //!< State is Enable
-    COMPONENT_RESET,                            //!< State is Reset
-    COMPONENT_MAX                               //!< End of enum
-}tvcomponent_state_t;
```

Comment:
Are tvcomponent_state_t, fp_mfrColorTemp_t, and input_source_offset_t supposed to be removed? This information is not present in the email conversations with Luc.

Reply:
All types in these ODM headers are marked as deprecated and supposed to be removed. I couldn't listing each type in the email, I provided a general description since the changes were just removal of deprecated types. Only for DolbyVision APIs, I have provided an explanation as it demanded some level of explanation in the way it shall be used for multiple modes. Please excuse if there was a lack of clarity.

Reply:
All types in these ODM headers are marked as deprecated and supposed to be removed. I couldn't listing each type in the email, I provided a general description since the changes were just removal of deprecated types. Only for DolbyVision APIs, I have provided an explanation as it demanded some level of explanation in the way it shall be used for multiple modes. Please excuse if there was a lack of clarity.

Reply:
Thank you for the clarification. Please let me know once your testing is complete.